src/libman/libman.csproj

```diff
@@ -2,16 +2,15 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFramework>netcoreapp2.1</TargetFramework>
+    <TargetFramework>$(NetCoreTFM)</TargetFramework>
     <RootNamespace>Microsoft.Web.LibraryManager.Tools</RootNamespace>
     <PackAsTool>true</PackAsTool>
-    <PackAsToolShimRuntimeIdentifiers>win-x64;win-x86;osx-x64</PackAsToolShimRuntimeIdentifiers>
     <GeneratePackageOnBuild>True</GeneratePackageOnBuild>
     <PackageId>Microsoft.Web.LibraryManager.Cli</PackageId>
     <Description>Command line tool for Library Manager</Description>
   </PropertyGroup>
   <ItemGroup>
-    <Reference Include="System.Runtime" />
+    <Reference Include="System.Runtime, Version=8.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" />
```

Comment:
how does this version get resolved in the future?  I.e. when updating to dotnet 10, will this break or does it get re-resolved to the version=10.0.0.0 assembly?